deploy/operator.yaml

```diff
@@ -16,13 +16,19 @@ spec:
       containers:
         - name: percona-server-mongodb-operator
           image: perconalab/percona-server-mongodb-operator:main
+          imagePullPolicy: Always
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /metrics
+              port: metrics
+              scheme: HTTP
           ports:
-          - containerPort: 60000
+          - containerPort: 8080
             protocol: TCP
             name: metrics
           command:
           - percona-server-mongodb-operator
-          imagePullPolicy: Always
```

Comment:
Why was it removed?

Reply:
It was not removed, it was just moved to sit below `image` since to me that makes more sense (and we have it in other operators in that place).